.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: Build
 
```

Comment:
In other connectors, do we use `ci.yml` as the build workflow name? If not, better to use build.yml or something like that

Reply:
It seems other connectors have also used same name as `ci.yml`